Fix UDMA not an integer on windows and add console output when patching UDMA

catogtp opened the pull request:
added sanity check/validation in TitanPatcher of the UDMA argument that sets the udma value to type int

added console output to the _patch_m8  self._udma

gaasedelen commented:
Thanks for the honest effort at fixing the issue, and it's a little embarrassing on my part. I added the argparse code last minute while cleaning things up and didn't fully vet the options once I switched things over :-)

catogtp commented:
> Thanks for the honest effort at fixing the issue, and it's a little embarrassing on my part. I added the argparse code last minute while cleaning things up and didn't fully vet the options once I switched things over :-)

No problem. I'm not ashamed to admit this is the first time I've ever worked in python so I'm glad a few of my tries were close!!
